README.markdown

```diff
@@ -11,6 +11,10 @@
     Online version
   </a>
   <span> | </span>
+  <a href="https://symfony.com/components">
+    Components docs
```

Comment:
```suggestion
    Components
```
?

Reply:
yes indeed as we are already in the doc repo
this wording has been taken from the main symfony repo :)